ironfish/src/rpc/routes/eth/blockNumber.ts

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { FullNode } from '../../../node'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type BlockNumberRequest = {} | undefined
+
+export const BlockNumberRequestSchema: yup.ObjectSchema<BlockNumberRequest> = yup
+  .object({})
```

Comment:
Will this ever take a parameter? Should never need one right? Maybe something like:
`yup.MixedSchema<void>`

Reply:
https://github.com/iron-fish/ironfish/blob/3c70456b30903f21bfdfe0c49d4e5f3d96fe9800/ironfish/src/rpc/routes/chain/getChainInfo.ts#L24-L26